components/layout_2020/display_list/mod.rs

```diff
@@ -69,6 +70,7 @@ pub struct WebRenderImageInfo {
 // webrender's `ItemTag` is private.
 type ItemTag = (u64, u16);
 type HitInfo = Option<ItemTag>;
+const INSERTION_POINT_LOGICAL_WIDTH: Au = Au(AU_PER_PX);
```

Comment:
No need to specify `: Au`. Also, `Au(AU_PER_PX)` is probably clearer as `Au::from_px(1)`?
Or even further, you are only using this for `INSERTION_POINT_LOGICAL_WIDTH.to_f32_px()`, so you could use `1.0` directly without dealing with `Au` at all.

Reply:
Actually I saw this from the old caret implementation and I thought there must be some reason behind this

https://github.com/servo/servo/blob/cf3ad2de774c8f8022135b0a9d466c197d961aa2/components/layout/display_list/builder.rs#L520

You're right about the potential of not needing to deal with `Au` at all though.